src/collective/cover/tiles/edit.py

```diff
@@ -65,8 +63,6 @@ def handleSave(self, action):
             old_data[item] = data[item]
         dataManager.set(old_data)
 
-        # notify about modification
-        notify(ObjectModifiedEvent(tile))
```

Comment:
No need to notify here, since ``dataManager.set`` method already notify the changes.